.mailmap

```diff
@@ -0,0 +1,38 @@
+Jan Nieuwenhuizen <janneke@gnu.org> <Jan Nieuwenhuizen janneke@gnu.org>
+Jan Nieuwenhuizen <janneke@gnu.org> <janneke@esur.localdomain>
+Jan Nieuwenhuizen <janneke@gnu.org> <janneke@janneke.ddns.htc.nl.philips.com>
+Jan Nieuwenhuizen <janneke@gnu.org> <janneke@peder.flower>
+Jan Nieuwenhuizen <janneke@gnu.org> janneke <janneke@gnu.org>
+
+Han-Wen Nienhuys <hanwen@lilypond.org> <hanwen@xs4all.nl>
+Han-Wen Nienhuys <hanwen@lilypond.org> hanwen <hanwen@lilypond.org>
+Han-Wen Nienhuys <hanwen@lilypond.org> hanwen <hanwen@xs4all.nl>
+Han-Wen Nienhuys <hanwen@lilypond.org> lilytest <hanwen@xs4all.nl>
+
+Boris-Chengbiao Zhou <bobo1239@web.de>
+Carl Sorensen <c_sorensen@byu.edu>
+Colin Campbell <colinpkcampbell@gmail.com>
+Colin Hall <colinghall@gmail.com>
+Eddy Pronk <epronk@muftor.com>
```

Comment:
IIUC "Eddy Pronk" refers to Jan Nieuwenhuizen, doesn't it?

Reply:
Does it? I was not under that impression.

Given a small amount of searching, Eddy Pronk authored two commits in GUB (aaa3311ac14cbb24ccb5ed55d913827405b300af and 8ff38da0a7013a891de18a0b7bec12b9d1fa6637) one of which (8ff38da0a7013a891de18a0b7bec12b9d1fa6637) was committed by Jan Nieuwenhuizen. Further, upon the deletion of the original GUB repository (https://github.com/janneke/gub) GitHub chose Eddy Pronk's fork (https://github.com/epronk/gub) as the new original repository.

As it is not clear, I would be inclined to leave Eddy Pronk as a separate entry unless Jan Nieuwenhuizen confirms that Eddy Pronk is an alias.

Reply:
On the off chance that it will help clear this up, can you clarify @epronk?

Reply:
It seems to me that Eddy Pronk is a replacement name for keeping the repository while allowing the removal of the account that owns it. I may be mistaken, though.

Anyway, this should not prevent us from adding the mailmap you contributed.